tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,30 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process
+{
+	typeset -l i killpid=$1
+
+	#attempt to kill
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_must $KILL $killpid
+	fi
+	#verify killed worked
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_mustnot $KILL $killpid
+	fi
+}
+
+function start_io
+{
+	typeset -l i sizetouse=$1
+	typeset -l i blocksize=$2
+	typeset -l i filename=$3
+
+	#start some IO
+	#This used to be: $FILE_TRUNC -f $((64 * 1024 * 1024)) -b 8192 -c 0 -r $TESTDIR/$TESTFILE1 &
+	dd_string="$DD if=/dev/urandom iflag=fullblock bs=$blocksize count=$sizetouse of=$filename & "
+	log_note "$dd_string"
+	dd_string
```

Comment:
The test suite itself provides a very simple tool called `file_write` which can be used to generate an IO load with a simple pattern.  And since it part of the test suite we can make it do whatever we want and then have that functionality on all platforms.